build.sbt

```diff
@@ -1105,7 +1105,18 @@ lazy val `scio-tensorflow`: Project = project
       "com.spotify" %% "magnolify-tensorflow" % magnolifyVersion % Test,
       "org.slf4j" % "log4j-over-slf4j" % slf4jVersion % Test,
       "org.slf4j" % "slf4j-simple" % slf4jVersion % Test
-    )
+    ),
+    Compile / PB.protoSources += target.value / s"metadata-$tensorFlowMetadataVersion",
+    Compile / PB.unpackDependencies := {
+      val tfMetadata = new URL(
+        s"https://github.com/tensorflow/metadata/archive/refs/tags/v$tensorFlowMetadataVersion.zip"
+      )
+      val protoFiles = IO.unzipURL(tfMetadata, target.value, "*.proto")
```

Comment:
do we want to future-proof this a little by adding the TF version to the file filter (i.e. `/**/v0/*.proto`)?

Reply:
sounds good 👍